ksql-parser/src/test/java/io/confluent/ksql/parser/SqlFormatterTest.java

```diff
@@ -148,5 +226,17 @@ public void shouldFormatOuterJoinWithoutJoinWindow() {
     final String expected = "left L\nFULL OUTER JOIN right R ON (('left.col0' = 'right.col0'))";
     assertEquals(expected, SqlFormatter.formatSql(join));
   }
+
```

Comment:
Should we add a case with a struct dereference here?

Reply:
We don't support non-column values in the join criteria yet.